packages/interbit-platform-tools/index.js

```diff
@@ -0,0 +1,73 @@
+const generateServiceUrl = (name, port) => {
+  console.log('name:', name)
+  const host = window.location.host
+  // const map =
+  console.log('host', host)
+  if (host.includes('localhost')) {
+    return `http://localhost:${port}`
+  }
+  if (name === 'interbit.io') {
+    if (!host.includes('herokuapp')) {
+      return 'https://interbit.io'
+    }
+  }
+  if (host.includes('interbit.io')) {
+    return `https://${name}.test-interbit.io`
+  }
+  if (host.includes('test-interbit.io')) {
+    return `https://${name}.test-interbit.io`
+  }
+
+  const serverName = herokuMap[name] || 'app-interbit-io'
+  const stage = getStage(host)
+  return `https://ib-${stage}-${serverName}.herokuapp.com`
+}
+
+const getStage = host => {
+  const stage = host.split('-')[1]
+  console.log('stage', stage)
+  return stage
+}
+
+const herokuMap = {
+  accounts: 'account',
+  store: 'app-store',
+  projects: 'app-projects',
+  'interbit.io': 'app-interbit-io'
+}
+
+// const localMap = {
```

Comment:
Old dns paths?

Reply:
This was merged in from Karin's branch and she's taken care of it there. It will disappear on another branch and be merged in there.